dashboard/test/fixtures/config/scripts_json/test-migrated-models.script_json

```diff
@@ -0,0 +1,133 @@
+{
+  "script": {
+    "name": "test-migrated-models",
+    "wrapup_video_id": null,
+    "hidden": true,
+    "login_required": false,
+    "properties": {
+      "peer_reviews_to_complete": 0,
+      "is_migrated": true
+    },
+    "new_name": null,
+    "family_name": null,
+    "seeding_key": {
+      "script.name": "test-migrated-models"
+    }
+  },
+  "lesson_groups": [
+    {
+      "key": "lesson group",
+      "user_facing": true,
+      "position": 1,
+      "properties": {
+        "display_name": "my lesson group"
+      },
+      "seeding_key": {
+        "lesson_group.key": "lesson group",
+        "script.name": "test-migrated-models"
+      }
+    }
+  ],
+  "lessons": [
+    {
+      "key": "new lesson",
+      "name": "My Lesson",
+      "absolute_position": 1,
+      "lockable": false,
+      "relative_position": 1,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "new lesson",
+        "lesson_group.key": "lesson group",
+        "script.name": "test-migrated-models"
+      }
+    }
+  ],
+  "lesson_activities": [
+    {
+      "key": "be74ab15-4657-426f-8935-65c9d285c61c",
+      "position": 1,
+      "properties": {
+        "name": "My Activity"
+      },
+      "seeding_key": {
+        "lesson_activity.key": "be74ab15-4657-426f-8935-65c9d285c61c",
+        "lesson.key": "new lesson",
+        "lesson_group.key": "lesson group",
+        "script.name": "test-migrated-models"
+      }
+    }
+  ],
+  "activity_sections": [
+    {
+      "key": "6174efd3-f122-476e-8fd3-8de6917bb8dc",
+      "position": 1,
+      "properties": {
+        "name": "My Activity Section",
+        "description": "My activity section description"
+      },
+      "seeding_key": {
+        "activity_section.key": "6174efd3-f122-476e-8fd3-8de6917bb8dc",
+        "lesson_activity.key": "be74ab15-4657-426f-8935-65c9d285c61c"
+      }
+    }
+  ],
+  "script_levels": [
+    {
+      "chapter": 1,
+      "position": 1,
+      "activity_section_position": 1,
+      "assessment": false,
+      "properties": {
+        "progression": "My Activity Section"
+      },
+      "named_level": null,
+      "bonus": false,
+      "seeding_key": {
+        "script_level.level_keys": [
+          "test_maze_level"
+        ],
+        "lesson.key": "new lesson",
+        "lesson_group.key": "lesson group",
+        "script.name": "test-migrated-models",
+        "activity_section.key": "6174efd3-f122-476e-8fd3-8de6917bb8dc"
+      },
+      "level_keys": [
+        "test_maze_level"
+      ]
+    }
+  ],
+  "levels_script_levels": [
+    {
+      "seeding_key": {
+        "level.key": "test_maze_level",
+        "script_level.level_keys": [
+          "test_maze_level"
+        ],
+        "lesson.key": "new lesson",
+        "lesson_group.key": "lesson group",
+        "script.name": "test-migrated-models",
+        "activity_section.key": "6174efd3-f122-476e-8fd3-8de6917bb8dc"
+      }
+    }
+  ],
+  "resources": [
```

Comment:
Why no resources?

Reply:
Oh I suppose it's because of the chicken/egg issue you mentioned

Reply:
yes, kind of. I could have added is_course, version_year, and family_name and then resources should work. that sidesteps the chicken/egg issue, which only applies to unit groups, not "standalone course" / is_course scripts. I didn't do that because I didn't think I needed to test every part of the new system, since we have other unit tests covering that, and this is more of an integration test. I could see value in having more "integration" test cases here covering those two ways of having course versions, though.

Reply:
Yeah I don't think it's necessary, was just wondering why it was the thing that wasn't being tested